### PR TITLE
Adjust spec to make clear inheritance from generic class is possible

### PR DESCRIPTION
**Pull request by mppf**
I'm not sure that the previous writing in the spec ruled it out,
but here I modify the spec to affirm that it is possible and
to describe a little bit what happens.

Reviewed by @vasslitvinov.

### CODE REVIEW

File: spec/Classes.tex [hunk 1]
```diff
@@ -58,7 +59,8 @@ \section{Class Declarations}
 Inheritance is described in~\rsec{Inheritance}.
 
 \begin{future}
```
Comment (vasslitvinov) [inline spec/Classes.tex]: I do not like how this future is written altogether. Also, what it describes can hardly be called "inheritance".

I would rather it be called "Open Issue" and contain just one sentence like "Classes that inherit from records are an area for future work."

Comment (vasslitvinov) [inline spec/Classes.tex]: Short of that, the proposed change is OK.

File: spec/Classes.tex [hunk 2]
```diff
@@ -106,7 +108,9 @@ \subsection{Class Types}
 \end{syntax}
```
Comment (vasslitvinov) [inline spec/Classes.tex]: I'd rather avoid repetition in the following. For example:

A class type, including a generic class type that is not fully instantiated, may appear in inheritance lists of other class declarations.

Also, is "fully specified" a legit term here? I am proposing "fully instantiated" instead.

Comment (mppf) [inline spec/Classes.tex]: Fully specified is the term the spec used in the "Class Types" section. I'm just trying to fit in.
See a whole section 15.1.1 and let me know which term you prefer after reviewing it. I will change to either.
It just says "fully specified" 2 lines up.

I'm happy to adjust the wording as you describe - we just need to choose "fully specified" or "fully instantiated".

Comment (vasslitvinov) [inline spec/Classes.tex]: Yes you are right. "fully specified" it is.

**Pull request merged**